descope/sdk/auth.go

```diff
@@ -152,7 +152,7 @@ type Password interface {
 	// redirectURL is an optional parameter that is used by Magic Link or Enchanted Link
 	// if those are the chosen reset methods. See the Magic Link and Enchanted Link sections
 	// for more details.
-	// templateOptions is used to pass dynamic options for the messaging (email / text message) template
+	// templateOptions is used to pass dynamic options for the messaging (email / text message, voice call) template
```

Comment:
user `/` to be consistent with the previous part of the sentence

Reply:
done